backend/app/views/spree/admin/users/_lifetime_stats.html.erb

```diff
@@ -8,19 +8,19 @@
     <table class="table" id="user-lifetime-stats" data-hook>
       <tr>
         <td width="30%"><%= Spree.t(:total_sales) %>:</td>
-        <td><%= @user.display_lifetime_value.to_html %></td>
```

Comment:
@tomurb please add `store` argument to these methods, please do not use complex queries inside views

Reply:
Done. The specs will require some fixes. I'll do it when I'll have more time.